staubli_tx2-90_support/tests/roslaunch_test.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<launch>
+
+  <group ns="load_tx2-90l__">
+    <include file="$(find staubli_tx2-90_support)/launch/load_tx2-90l.launch"/>
+  </group>
+
+  <group ns="test_tx2-90l__">
+    <include file="$(find staubli_tx2-90_support)/launch/test_tx2-90l.launch"/>
+  </group>
+
+</launch>
```

Comment:
Could you please create separate test files for each variant?

See [fanuc_m6ib_support/tests](https://github.com/ros-industrial/fanuc/tree/284429d062220f9b3e58b680d29ec20f9555bdf3/fanuc_m6ib_support/tests) for examples.